packages/frontend/src/renderer/components/widgets/performCommunityAction/PerformCommunityAction.test.tsx

```diff
@@ -114,7 +114,7 @@ describe('PerformCommunityAction component (join community mode)', () => {
       community={false}
     />
     const result = renderComponent(component)
-    const textInput = result.queryByPlaceholderText('Invite link')
+    const textInput = result.queryByPlaceholderText('Invite code')
```

Comment:
Could you use `inviteLinkField.fieldProps.placeholder` here instead of hardcoded string?